src/main/java/com/google/devtools/build/lib/skyframe/serialization/autocodec/Initializers.java

```diff
@@ -61,6 +61,7 @@ static TypeSpec.Builder initializeCodecClassBuilder(
             AnnotationSpec.builder(ClassName.get(SuppressWarnings.class))
                 .addMember("value", "$S", "unchecked")
                 .addMember("value", "$S", "rawtypes")
+                .addMember("value", "$S", "SunApi")
```

Comment:
Where does this come from?

Reply:
this was a workaround to an internal test regarding serialization. no longer needed